pkg/interface/src/views/landscape/components/Participants.tsx

```diff
@@ -297,7 +297,13 @@ function Participant(props: {
 
   const avatar =
     contact?.avatar !== null && !hideAvatars ? (
-      <img src={contact.avatar} height={32} width={32} className="dib" />
+      <img 
```

Comment:
`display="inline-block"` actually won't work with the unstyled `img` element. What you'll want is to add an `Image` import from indigo-react (you can find the indigo elements we import at the top), then just change this to `<Image`. That's all. You can append it as a review comment — `landscape: address #4517 feedback`

Reply:
Thank you for your patience.